BraveShared/de.lproj/BraveShared.strings

```diff
@@ -190,9 +190,6 @@
 /* Brave Rewards settings title */
 "BraveRewardsTitle" = "Brave Rewards";
 
-/* Section title for adbblock, tracking protection, HTTPS-E, and cookies */
-"BraveShieldDefaults" = "Brave Shield-Standardeinstellungen";
```

Comment:
This string isn't used anymore, can't find it in code

Reply:
It's renamed to `Shields_Defaults` now, will be pulled from next translations PR